dvl-a50/blueoshelper.py

```diff
@@ -29,7 +30,7 @@ def post(url, data):
             return response.read()
 
     except Exception as error:
-        print("Error in request: {0}: {1}".format(url, error))
-        print(error.read())
-        print(data)
+        logger.warning("Error in request: {0}: {1}".format(url, error))
+        logger.warning(error)
```

Comment:
This logs the error twice, right?
```suggestion
        logger.warning(f"Error in request: {url}: {error}")
```

Reply:
oops